Remotion/Web/Core/Res/Themes/NovaViso/HTML/Style.css

```diff
@@ -35,14 +35,35 @@ Black
 
 /* Basic styles used to define the font, text-size, etc. */
 
+@font-face {
+  font-family: 'Roboto-Medium';
+  src: url('../../../Html/Roboto-Medium.ttf') format('truetype');
```

Comment:
Okay, so we can only import it like this? "good" (as in, okay, we have to do it this way then and deal with the consequences).

So, since we cannot include the font from the code-behind but only via the CSS file, we can now move the TTF files into the themes html folder isntead of the shared one. Reasoning: we no longer have to do a global registration. Also, things get easier, no changes in global.asax, etc.